pkg/operations/kubernetes162upgrader.go

```diff
@@ -64,14 +66,21 @@ func (ku *Kubernetes162upgrader) RunUpgrade() error {
 	upgradeMasterNode.ResourceGroup = ku.ClusterTopology.ResourceGroup
 	upgradeMasterNode.Client = ku.Client
 
-	// for _, vm := range *ku.ClusterTopology.MasterVMs {
-	// 1.	Shutdown and delete one master VM at a time while preserving the persistent disk backing etcd.
-	upgradeMasterNode.DeleteNode()
-	// 2.	Call CreateVMWithRetries
-	upgradeMasterNode.CreateNode(loopCount)
-	loopCount++
+	// Sort by VM Name (e.g.: k8s-master-22551669-0) offset no. in descending order
+	sort.Sort(sort.Reverse(armhelpers.ByVMNameOffset(*ku.ClusterTopology.MasterVMs)))
```

Comment:
Because if you see the sort implementation for Go (sort.Interface) it only has Less function that needs to be implemented. So basically you can only sort ascending and the only way I could find to sort descending is by reversing the sorted list.